package/scripts/params.py

```diff
@@ -73,15 +72,18 @@
 else:
     hdfs_namespace = map_cdap_site['hdfs.namespace']
 
+cdap_security_enabled = config['configurations']['cdap-site']['security.enabled']
+
 # Kerberos stuff
+kerberos_enabled = config['configurations']['cluster-env']['security_enabled']
```

Comment:
I renamed `secuity_enabled` to `kerberos_enabled` to attempt to remove confusion. Ambari uses the term "security enabled" to mean Kerberos. It is a global cluster config named `security.enabled` so you can see how our choice of the same can be confusing. I wanted to differentiate between that and CDAP Security, which is `cdap_security_enabled` and allow them to be used separately. I'll need `cdap_security_enabled` for anything that will need to fetch a token (like tx-debugger), but it's currently unused.